### PR TITLE
added validation for registration fields

### PR DESCRIPTION
**Pull request by LarisaBothaza**

### CODE REVIEW

**Comment by DascaluCosmin** [comment]
**As discussed, from now on, we add the entire BE team for Review. It's not required for all members to review, but at least 1 should review the code.**

File: src/main/java/com/sintern/domain/entity/User.java [hunk 1]
```diff
@@ -22,8 +23,14 @@
     @Column(columnDefinition = "uuid")
     UUID id;
 
+    @NotNull(message = "Email is required")
+    @NotBlank(message = "Email is required")
+    @Email(message = "Invalid email format")
     @Column(name = "email", unique = true)
+    @Size(min = 7, max = 74, message = "Email should have between 7 and 74 characters")
```
Comment (DascaluCosmin) [inline src/main/java/com/sintern/domain/entity/User.java:30]: I think the **email** should have a **REGEX** as well.

Comment (LarisaBothaza) [reply src/main/java/com/sintern/domain/entity/User.java:30]: My mistake, I thought the **@Email** annotation was enough.

**Review by meOtniel — APPROVED** [review]
Good job!

**Review by DascaluCosmin — APPROVED** [review]

**Pull request merged**